systemuser.sh

```diff
@@ -31,19 +31,23 @@ mkdir -p $JPY_DIR
 JPY_LOCAL_DIR=$SCRATCH_HOME/.local
 mkdir -p $JPY_LOCAL_DIR
 export JUPYTER_CONFIG_DIR=$JPY_DIR
-# Our kernels will be in $JPY_LOCAL_DIR/share/jupyter, $LCG_VIEW/share/jupyter is needed for the notebook extensions
-export JUPYTER_PATH=$JPY_LOCAL_DIR/share/jupyter:$LCG_VIEW/share/jupyter:$EXTRA_LIBS
-export KERNEL_DIR=$JPY_LOCAL_DIR/share/jupyter/kernels
+JUPYTER_LOCAL_CONFIG_DIR=$JPY_LOCAL_DIR/share/jupyter
```

Comment:
Perhaps this should be called JUPYTER_LOCAL_PATH?
http://jupyter.readthedocs.io/en/latest/projects/jupyter-directories.html